src/app/public/modules/repeater/repeater.component.spec.ts

```diff
@@ -532,6 +532,29 @@ describe('Repeater item component', () => {
         expect(fixture.nativeElement).toBeAccessible();
       });
     }));
+
+    it('update the isSelected property on the input model', fakeAsync(() => {
+      let fixture = TestBed.createComponent(RepeaterTestComponent);
+      let cmp: RepeaterTestComponent = fixture.componentInstance;
+      fixture.detectChanges();
+
+      tick();
+
+      cmp.repeater.items.forEach(item => item.selectable = true);
+
+      const repeaterItems = cmp.repeater.items.toArray();
+      repeaterItems[2].updateIsSelected({source: undefined, checked: true});
```

Comment:
While `updateIsSelected` is certainly changing the model, would you mind triggering this logic from a click event? Sometimes, there are obscure bugs and timing issues hiding along the way with user interaction. If we test this logic just as a user would, we'd have a better chance of exposing any of those weird errors.

Reply:
Good call, went ahead and updated it to use a click event instead.